sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/batch_deployment.py

```diff
@@ -264,6 +275,8 @@ def _from_rest_object(cls, deployment: BatchDeploymentData):  # pylint: disable=
             max_concurrency_per_instance=deployment.properties.max_concurrency_per_instance,
             endpoint_name=_parse_endpoint_name_from_deployment_id(deployment.id),
             properties=deployment.properties.properties,
+            creation_context=SystemData._from_rest_object(deployment.system_data),
```

Comment:
This line for creation context, is fixing a different bug, unrelated to tags right?

Reply:
Yes, we need to add the creation_context as well. Let me update the PR title.